packages/cli/tests/unit/depends.spec.ts

```diff
@@ -10,7 +10,7 @@ import { listAppMapFiles, verbose } from '../../src/utils';
 if (process.env.DEBUG !== 'true') tmp.setGracefulCleanup();
 
 const fixtureDir = join(__dirname, 'fixtures', 'ruby');
-const now = Date.now();
+const now = Date.now() / 1000;
```

Comment:
I don't understand this change.

Reply:
[Apparently, `utimesSync` takes timestamps in seconds, but `Date.now()` generates them in milliseconds](https://github.com/nodejs/node/issues/33227#issuecomment-623391936). But I'm confused as to why it works in Linux / Mac if that's the case...

Reply:
I agree, I implemented this